packages/dso-toolkit/components/Componenten/icon/icon.config.js

```diff
@@ -3,11 +3,7 @@ const getIcons = require('../../../utils/get-icons');
 const icons = getIcons();
 
 module.exports = {
-  label: 'Icon (Web Component)',
-  meta: {
-    webComponent: 'dso-icon',
-    markup: false
-  },
+  label: 'Icon (Web Component) 🚀', // Fake the web component rocket indicator, Stencil's Hydrate function removes <style> elements which contain icon table styling.
```

Comment:
Toelichting: doordat `meta.webComponent` weg is, wordt dit component niet meer verwerkt door de hydrate functie (en behouden we de tabel), maar zie je ook geen raketje meer, die staat nu dus geforceerd in de titel zelf.